api-tester/src/main/java/com/revenuecat/apitester/kotlin/OfferingAPI.kt

```diff
@@ -19,6 +19,7 @@ private class OfferingAPI {
             val weekly: Package? = weekly
             val p1: Package = offering[""]
             val p2: Package = getPackage("")
+            val metadata: Map<String, Any> = metadata
```

Comment:
I think we could try to add accessor for different types of values... Something like `getMetadataString`, `getMetadataInt`, `getMetadataLong`, `getMetadataBool`, `getMetadataDouble` and its optional versions `optMetadataString`,... This wouldn't support nested values, but could be more useful for more common cases.

This way we would support some easier access methods for Java. If we only provide the Map<String, Any>, we leave it to the developer to transform the `Any` to their desired type, which I don't think is a great API... Thoughts?

Alternatively, I would prefer to only support strings and just convert everything into a String, and have metadata be a `Map<String, String>`.